docs/pkgdown.yml

```diff
@@ -1,8 +1,8 @@
-pandoc: 3.1.1
```

Comment:
Is this change auto-generated? Just curious about why the pandoc version is in quotes now

Reply:
I noticed that too. Yes, auto. I just figured it's fixing something.